src/components/revaluation/ReservationRevaluationInfo.module.scss

```diff
@@ -0,0 +1,42 @@
+.tooltip {
```

Comment:
Was there a tooltip for this component?

Reply:
Lets say "there was an attempt", but it really didn't work in the list view as expected. So yeah, leftover trash.